docs/scanning.rst

```diff
@@ -24,6 +24,10 @@ To discover Bluetooth devices that can be connected to:
 
 .. warning:: Do not name your script `bleak.py`! It will cause a circular import error.
 
+.. warning:: On macOS you need to give your terminal permission to access Bluetooth.
```

Comment:
Some terminal programs are configured correctly for Bluetooth, so this should probably say "you *may* need to ..."

Reply:
Fair enough, will make the edit now